library/core/class.oauth2.php

```diff
@@ -466,6 +470,10 @@ protected function realAuthorizeUri(array $state = []): string {
         $state['token'] = $this->ssoUtils->getStateToken();
         $get['state'] = $this->encodeState($state);
 
+        if (array_key_exists('Prompt', $provider) && isset($provider['Prompt'])) {
+            $get['Prompt'] = $provider['Prompt'];
```

Comment:
By passing the `prompt` as `Prompt` it didn't work with my Authentication provider. It is case sensitive. It needs to be `prompt`.